src/main/java/net/schmizz/sshj/transport/cipher/BaseCipher.java

```diff
@@ -38,14 +38,20 @@
     }
 
     private final int ivsize;
+    private final int authSize;
     private final int bsize;
     private final String algorithm;
     private final String transformation;
 
     private javax.crypto.Cipher cipher;
```

Comment:
Let's make `cipher` protected so that it's accessible without a public `getCipherInstance` method. We should not leak it to the world.